libqtile/window.py

```diff
@@ -1053,39 +1054,19 @@ def toscreen(self, index=None):
                 raise CommandError('No such screen: %d' % index)
         self.togroup(screen.group.name)
 
-    def match(self, wname=None, wmclass=None, role=None):
+    def match(self, match):
```

Comment:
Since this barely does more than `match.compare(win)`, we could maybe deprecate the entire function.

Reply:
+1, does the internal API use it at all? I'd say just drop it if not. But not necessary for this PR.

Reply:
I didn't find any places using the function (other than `Window.cmd_match`). I'll do this in another PR, then.